themes-default/slim/views/manage_backlogOverview.mako

```diff
@@ -144,13 +144,14 @@ const startVue = () => {
                 </tr>
                 % for cur_result in showSQLResults[(cur_show.indexer, cur_show.series_id)]:
                     <%
-                        old_status, old_quality = Quality.split_composite_status(cur_result['status'])
-                        archived_status = Quality.composite_status(ARCHIVED, old_quality)
+                        old_status = cur_result['status']
+                        old_quality = cur_result['quality']
+                        archived_status = ARCHIVED
```

Comment:
We can remove it. The only place this is used in is line 175:
https://github.com/pymedusa/Medusa/blob/942eb3095ace01860979f1f6caea4eae1928c925/themes-default/slim/views/manage_backlogOverview.mako#L175

So we can just substitute it with `ARCHIVED`.